gpt_engineer/steps.py

```diff
@@ -218,11 +218,17 @@ def gen_code_after_unit_tests(ai: AI, dbs: DBs) -> List[dict]:
 def execute_entrypoint(ai: AI, dbs: DBs) -> List[dict]:
     command = dbs.workspace["run.sh"]
 
-    print("Do you want to execute this code?")
+    print()
+    print(
+        colored(
+            "Do you want to execute this code? (y/n)",
```

Comment:
I thought this is an important decision and to better see it above the printed execution command below, we paint it in red. Plus, we add the (y/n) which is also used for the other upcoming questions.